README.md

```diff
@@ -2,14 +2,21 @@
 # io-pay
 - [io-pay](#io-pay)
   - [Prerequisites](#prerequisites)
+  - [Documentation](#documentation)
   - [Running](#running)
+  - [Sequence diagram : _WISP light_](#sequence-diagram--wisp-light)
   
 Is a [WISP](https://docs.italia.it/italia/pagopa/pagopa-specifichepagamenti-docs/it/stabile/_docs/SANP_2.2_Sez2_Cap06_ComponentiTecnicheNodo.html#componente-wisp) replacement project and it will be the first web interface to allow guest (_citizen_) payment of [pagoPa](https://www.pagopa.gov.it/) system.
 
 ## Prerequisites
 
 - [yarn](https://classic.yarnpkg.com/en/docs/getting-started)
 
+## Documentation
```

Comment:
@pasqualespica Theese documents are private, I don't know if they can be shared here

Reply:
Thanks you're right @GiovanniMancini. I think that a `public` repo project as `io-pay` should have a `public` documentation too.  I don't know if we can change the currently visibility of our documnetation to `public` or crate a new one. Could be a point ti discuss into next review planning ?